python/setup.py

```diff
@@ -88,7 +88,7 @@ def build_extension(self, ext):
         if not os.path.exists(llvm_build_dir):
             os.makedirs(llvm_build_dir)
         # python directories
-        python_include_dirs = [distutils.sysconfig.get_python_inc()] + ['/usr/local/cuda/include']
+        python_include_dirs = [distutils.sysconfig.get_python_inc()] + ['/usr/local/cuda/include'] + ['/opt/rocm/include']
```

Comment:
This will likely need to be conditionalized on ROCm/CUDA using TRITON_USE_ROCM env variable.

Reply:
Done